Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.8-slim
+ 
+WORKDIR /app
+
+COPY get-poetry.py ./
+
+RUN python get-poetry.py --yes && \
+    rm get-poetry.py && \
+    . "$HOME/.poetry/env" && \
+    poetry config virtualenvs.create false
+
+COPY pyproject.toml poetry.lock ./
+
+RUN . "$HOME/.poetry/env" && \
+    poetry install --no-root
+
+COPY epimodel luigi.cfg logging.conf ./
```

Comment:
If you are using the docker just as an environment, i would not even copy any sources or anything, all that should be mounted. It just causes an unnecessary docker rebuild when you change sources.

Reply:
Agree, although I hoped that we could use it to e.g. run tests inside CI.